### ISSUE TITLE
Feature Request: Linux app please

### ISSUE THREAD

**Issue by mohammadrafigh** [gilbitron/Splashify#1]
it would be great to have a Linux (at least Ubuntu) version of Splashify. since it's an electron app it would be easy to build one...

**Comment by gilbitron** [comment]
I've added AppImage and rpm packages to the [latest release](https://github.com/gilbitron/Splashify/releases/tag/v1.1.0)

**Comment by gilbitron** [comment]
Actually I've just realised that the module I'm using for setting the wallpaper doesn't have a linux version, so I'm afraid I've had to remove the linux builds as they won't work.

**Comment by mohammadrafigh** [comment]
Yes I tested, it just saves the picture in pictures folder. I may look into codes if you are ok

**Comment by gilbitron** [comment]
Please do. This is the module https://github.com/sindresorhus/wallpaper. It looks like it should support Linux, so not sure why it's not working. It's possible that the image processing library is failing https://github.com/oliver-moran/jimp. Will need to do some testing.

**Comment by mohammadrafigh** [comment]
there is an open [issue](https://github.com/sindresorhus/wallpaper/issues/18) and a [PR](https://github.com/sindresorhus/wallpaper/pull/17) for that, it seems "wallpaper" module has some problems currently with some desktop environments like KDE, Gnome 2, Cinnamon... which desktop environment did you test with? I use Gnome 3 and no result... it seems we should wait for next release of "wallpaper".

**Comment by gilbitron** [comment]
I tested with Ubuntu. Will wait for the next release to see if a fix is included.
